conf/broker.conf

```diff
@@ -258,7 +258,7 @@ brokerDeduplicationEntriesInterval=1000
 brokerDeduplicationProducerInactivityTimeoutMinutes=360
 
 # When a namespace is created without specifying the number of bundle, this
-# value will be used as the default
+# value will be used as the default，the maximum number of namespaces is 2147483648.
```

Comment:
Does this `2147483648` actually works for bundle number?

Reply:
theoretically，not actually

Reply:
So this info is misleading. What about like "int type, maximum number depends on metadata performance, normally not larger than ???"

Reply:
@hangc0276 please take a look.